tests/data/iceberg/default.db/iceberg_table_2/README.md

```diff
@@ -0,0 +1,32 @@
+This table was generated with the following code:
```

Comment:
Could also mention that we manually renamed the metadata version files to comply with the FS-based convention.